dashboard/config/locales/authored_hints.es-MX.yml

```diff
@@ -442,8 +442,8 @@ es-MX:
         Course_4_Artist_12_b: 'La forma de la unidad es casi la misma que en el último
           acertijo, excepto que el medio hexágono es un poco más grande. '
         Course_4_Artist_12_c: "1.  Las primeras tres líneas tienen 60 píxeles de largo
-          y necesitas girar 60 grados.\n     2) Las siguientes dos líneas tienen 80
-          píxeles de largo y necesitas girar 120 grados "
+          y necesitas girar 60 grados.\n     2\\) Las siguientes dos líneas tienen
```

Comment:
Is the number 2 or paren escaped?  Not sure where there are two backward slashes.

Reply:
The paren; note this string is serialized in yaml, so you need to escape your slashes, too